### ISSUE TITLE
Not dying when getting squished

### ISSUE THREAD

**Issue by neomoth** [ExOK/Celeste64#29]
The game doesn't seem to account for getting stuck inside of collision, wondering if this might be intentional but then again it leads to stuff like this which feels blatantly wrong.
https://github.com/ExOK/Celeste64/assets/69372252/7da6ba38-970b-46be-935c-e0dd0c7ea4ea
To me you should be getting squished and killed here like you would in the original game.
Is this an error or is this intentional?

**Comment by NoelFB** [comment]
Yeah this should have probably killed you ... but there are people using it for various exploits/grabless runs now so I'm hesitant to fix it haha. There are scenarios like that one specifically that aren't great.

**Comment by neomoth** [comment]
I think there's probably some way it could be reworked into a mechanic but have it kill you under normal circumstances perhaps. No idea what that mechanic would look like, but it's just a thought.

**Comment by NoelFB** [comment]
Yeah, I think we're probably going to fix squishing and make grabless possible a different way? It feels like a general bug that would be good to fix.
